corehq/apps/es/es_query.py

```diff
@@ -302,6 +302,31 @@ def remove_default_filter(self, default):
             query._default_filters.pop(default)
         return query
 
```

Comment:
Actually, since I'm reopening as a different PR, can you comment on https://github.com/dimagi/commcare-hq/pull/9365/files#diff-6bc5251ac254d50d65a8d33297cdd7fcR302 instead?